test/UnitTests/TestFramework.UnitTests/Attributes/DataRowAttributeTests.cs

```diff
@@ -164,4 +178,22 @@ public void GetDisplayNameForMultipleArraysOfMultipleItems()
         // Assert
         Verify(displayName == "MyMethod (System.String[],System.String[])");
     }
+
+    private class DummyDataRowAttribute : DataRowAttribute
+    {
+        public DummyDataRowAttribute(object data)
+            : base(data)
+        {
+        }
+
+        public override string GetDisplayName(MethodInfo methodInfo, object[] data)
+        {
+            if (!string.IsNullOrWhiteSpace(DisplayName))
+            {
+                return DisplayName;
+            }
+
+            return methodInfo.Name;
```

Comment:
Please return a constant string instead (and update assertion) so that we don't get confused with a change of behavior that would still use method name.